pkg/reconciler/pipelinerun/cancel_test.go

```diff
@@ -38,196 +38,131 @@ import (
+					TypeMeta:         runtime.TypeMeta{Kind: "CustomRun"},
+					Name:             "t2",
+					PipelineTaskName: "task-2",
+				}},
 			}},
 		},
 		customRuns: []*v1beta1.CustomRun{
 			{ObjectMeta: metav1.ObjectMeta{Name: "t1"}},
 		},
 	}, {
-		name:           "child-references-with-both",
-		embeddedStatus: config.BothEmbeddedStatus,
-		pipelineRun: &v1beta1.PipelineRun{
-			ObjectMeta: metav1.ObjectMeta{Name: "test-pipeline-run-cancelled"},
-			Spec: v1beta1.PipelineRunSpec{
-				Status: v1beta1.PipelineRunSpecStatusCancelled,
-			},
-			Status: v1beta1.PipelineRunStatus{PipelineRunStatusFields: v1beta1.PipelineRunStatusFields{
-				ChildReferences: []v1beta1.ChildStatusReference{
-					{
-						TypeMeta:         runtime.TypeMeta{Kind: "TaskRun"},
-						Name:             "t1",
-						PipelineTaskName: "task-1",
-					},
-					{
-						TypeMeta:         runtime.TypeMeta{Kind: "TaskRun"},
-						Name:             "t2",
-						PipelineTaskName: "task-2",
-					},
-					{
-						TypeMeta:         runtime.TypeMeta{Kind: pipeline.RunControllerName},
-						Name:             "r1",
-						PipelineTaskName: "run-1",
-					},
-					{
-						TypeMeta:         runtime.TypeMeta{Kind: pipeline.RunControllerName},
-						Name:             "r2",
-						PipelineTaskName: "run-2",
-					},
-				},
-			}},
-		},
-		taskRuns: []*v1beta1.TaskRun{
-			{ObjectMeta: metav1.ObjectMeta{Name: "t1"}},
-			{ObjectMeta: metav1.ObjectMeta{Name: "t2"}},
-		},
-		runs: []*v1alpha1.Run{
-			{ObjectMeta: metav1.ObjectMeta{Name: "r1"}},
-			{ObjectMeta: metav1.ObjectMeta{Name: "r2"}},
-		},
-	}, {
-		name:           "child-references-with-both-some-missing",
-		embeddedStatus: config.BothEmbeddedStatus,
-		pipelineRun: &v1beta1.PipelineRun{
-			ObjectMeta: metav1.ObjectMeta{Name: "test-pipeline-run-cancelled"},
-			Spec: v1beta1.PipelineRunSpec{
-				Status: v1beta1.PipelineRunSpecStatusCancelled,
-			},
-			Status: v1beta1.PipelineRunStatus{PipelineRunStatusFields: v1beta1.PipelineRunStatusFields{
-				ChildReferences: []v1beta1.ChildStatusReference{
-					{
-						TypeMeta:         runtime.TypeMeta{Kind: "TaskRun"},
-						Name:             "t1",
-						PipelineTaskName: "task-1",
-					},
-					{
-						TypeMeta:         runtime.TypeMeta{Kind: "TaskRun"},
-						Name:             "t2",
-						PipelineTaskName: "task-2",
-					},
-					{
-						TypeMeta:         runtime.TypeMeta{Kind: pipeline.RunControllerName},
-						Name:             "r1",
-						PipelineTaskName: "run-1",
-					},
-					{
-						TypeMeta:         runtime.TypeMeta{Kind: pipeline.RunControllerName},
-						Name:             "r2",
-						PipelineTaskName: "run-2",
-					},
-				},
-			}},
-		},
-		taskRuns: []*v1beta1.TaskRun{
-			{ObjectMeta: metav1.ObjectMeta{Name: "t1"}},
-		},
-		runs: []*v1alpha1.Run{
-			{ObjectMeta: metav1.ObjectMeta{Name: "r2"}},
-		},
-	}, {
-		name:           "child-references-with-minimal",
-		embeddedStatus: config.DefaultEmbeddedStatus,
+		name: "child-references",
```

Comment:
can you rename the test cases with "child-references" in the name to something more descriptive? All of the test cases now have child references, so putting it in the test name isn't very informative.